htdocs/core/class/discount.class.php

```diff
@@ -4,6 +4,7 @@
  * Copyright (C) 2024      Alexandre Janniaux   <alexandre.janniaux@gmail.com>
  * Copyright (C) 2024       Frédéric France             <frederic.france@free.fr>
  * Copyright (C) 2024		MDW							<mdeweerd@users.noreply.github.com>
+ * Copyright (C) 2024		Noé Cendrier		<noe.cendrier@altairis.fr>
```

Comment:
@eldy My email address is repelled by by pre-commit, and I don’t remember how we addressed this last the last time 😅